File: SoalB/Pyhton/RAM.py
class RAM():
    # atribut
    __priceRAM = 0
    __capacityRAM = 0

    # constructor
    def __init__(self):
        self.__priceRAM = 0
        self.__capacityRAM = 0

    # membuat get set method harga RAM
    def setPriceRAM(self, priceRAM):
        self.__priceRAM = priceRAM

    def getPriceRAM(self):
        return self.__priceRAM

    # membuat get set method kapasitas RAM
    def setCapacityRAM(self, capacityRAM):
        self.__capacityRAM = capacityRAM

    def getCapacityRAM(self):
        return self.__capacityRAM

    # menampilkan data ram
    def printRAM(self):
        print("RAM Price(IDR)       : " + str(self.getPriceRAM()))
        print("RAM Capacity(GB)     : " + str(self.getCapacityRAM()))
        print("---------------------------------------")

File: SoalA/C++/RAM.cpp
#include <iostream>
#include <string>

using namespace std;

class RAM
{
    /* membuat atribut */
private:
    int priceRAM;
    int capacityRAM;

public:
    /* constructor */
    RAM(int priceRAM, int capacityRAM)
    {
        this->priceRAM = priceRAM;
        this->capacityRAM = capacityRAM;
    }

    RAM()
    {
    }

    /* Membuat get set method harga RAM */
    void setPriceRAM(int priceRAM)
    {
        this->priceRAM = priceRAM;
    }
    int getPriceRAM()
    {
        return this->priceRAM;
    }

    /* Membuat get set method kapasitas RAM */
    void setCapacityRAM(int capacityRAM)
    {
        this->capacityRAM = capacityRAM;
    }
    int getCapacityRAM()
    {
        return this->capacityRAM;
    }

    /* menampilkan data DISK */
    void printRAM()
    {
        cout << "RAM Price(IDR)     : " << this->getPriceRAM() << endl;
        cout << "RAM Capacity(GB)   : " << this->getCapacityRAM() << endl;
        cout << "====================================================" << endl;
    }

    /* destructor */
    ~RAM()
    {
    }
};